docs/Dockerfile

```diff
@@ -5,7 +5,7 @@ MAINTAINER Mu Li <muli@cs.cmu.edu>
 # First, build MXNet binaries (ref mxnet/docker/cpu/Dockerfile)
 #
 
-RUN apt-get update && apt-get install -y build-essential git libopenblas-dev libopencv-dev
+RUN apt-get update && apt-get install -y build-essential git libopenblas-dev liblapack-dev libopencv-dev
```

Comment:
Did you mean Ubuntu 14.04 as the Host OS? Can we use Ubuntu 16.04 LTS as base Image for building the Docker and still run the Docker on Ubuntu 14.04?

If we have to stick with Ubuntu 14.04 LTS as base Image, effectively no official package provides OpenBLAS's LAPACK routines.